manifests/profile/base/apt.pp

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2018 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::base::apt
+#
+# Manage apt.
+#
+# @example
+#   include nebula::profile::base::apt
+class nebula::profile::base::apt {
+  # Ensure that apt knows to never ever install recommended packages
+  # before it installs any packages.
+  File['/etc/apt/apt.conf.d/99no-recommends'] -> Package<| |>
```

Comment:
In case anybody's curious, https://puppet.com/docs/puppet/4.10/lang_collectors.html